src/ui/App.jsx

```diff
@@ -1,17 +1,31 @@
 import { hot } from 'react-hot-loader';
+import {
+  HashRouter as Router,
```

Comment:
What was the idea behind using `HashRouter` for the routing rather than `BrowserRouter`?

Reply:
I sort of assumed BrowserRouter wouldn't work, but I didn't actually check.. :)